xbmc/pvr/addons/PVRClient.h

```diff
@@ -550,6 +556,14 @@ namespace PVR
     */
     PVR_ERROR GetRecordingEdl(const CPVRRecording& recording, std::vector<PVR_EDL_ENTRY>& edls);
 
+    /*!
+    * @brief Retrieve the size of a recording on the backend.
+    * @param recording The recording.
+    * @param sizeInBytes The size in bytes or -1 on error
```

Comment:
no need for special error value as the method signals errors using its return value.